Sources/AirwallexRisk/AirwallexRiskContext.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AirwallexRiskContext.swift
+//  AirwallexRisk
+//
+//  Created by Richie Shilton on 5/7/2023.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+class AirwallexRiskContext {
+    @Storage(key: AirwallexUserDefaultKey.user, defaultValue: .init())
```

Comment:
As much as i like property wrapper, it makes this class impossible to test

Reply:
Will leave, as discussed offline. Tested the integration.

Reply:
I've removed the decoration and will init the Storage objects in the Context init - so we can pass in the UserDefault suite explicitly.